executor/builder.go

```diff
@@ -1589,7 +1591,7 @@ func buildNoRangeIndexLookUpReader(b *executorBuilder, v *plan.PhysicalIndexLook
 		keepOrder:         is.KeepOrder,
 		desc:              is.Desc,
 		tableRequest:      tableReq,
-		columns:           is.Columns,
+		columns:           ts.Columns,
```

Comment:
why change this?

Reply:
Though `IndexScan`' `Columns` is directly copied from `DataSource`, but its actual meaning is not this.
So change to `TableScan`'s `Columns' to make this more clear.